server/analytics/analytics.go

```diff
@@ -25,7 +25,12 @@ type ClusterArgs struct {
 
 func OpenDB(dsn string) (*Analytics, error) {
 	if strings.Contains(dsn, "tls_config=") {
-		caCert, err := ioutil.ReadFile("../certs/clickhouse/ca/ca.crt")
+		configPath := "../certs/clickhouse/ca/ca.crt"
+		if strings.Contains(dsn, "sslrootcert=") {
```

Comment:
I would use [`url.Parse`](https://gobyexample.com/url-parsing) to extract query params from the DSN.
And I would suggest to name the parameter `ssl_root_cert`.